kura/org.eclipse.kura.wire.provider/src/main/java/org/eclipse/kura/internal/wire/WireServiceImpl.java

```diff
@@ -279,43 +314,43 @@ public void deleteWireConfiguration(final WireConfiguration wireConfiguration) {
      *             if argument is null
      */
     private void extractProperties(final Map<String, Object> properties) {
-        requireNonNull(properties, s_message.propertiesNonNull());
-        s_logger.debug(s_message.exectractingProp());
+        requireNonNull(properties, message.propertiesNonNull());
+        logger.debug(message.exectractingProp());
         // clear the configurations first
-        if (this.options != null) {
-            final List<WireConfiguration> list = this.options.getWireConfigurations();
+        if (this.wireServiceOptions != null) {
+            final List<WireConfiguration> list = this.wireServiceOptions.getWireConfigurations();
             list.clear();
         }
-        this.options = WireServiceOptions.getInstance(properties);
+        this.wireServiceOptions = WireServiceOptions.getInstance(properties);
         this.properties = properties;
-        for (final WireConfiguration conf : this.options.getWireConfigurations()) {
+        for (final WireConfiguration conf : this.wireServiceOptions.getWireConfigurations()) {
             this.wireConfigs.add(conf);
         }
-        s_logger.debug(s_message.exectractingPropDone());
+        logger.debug(message.exectractingPropDone());
     }
 
     /** {@inheritDoc} */
     @Override
     public ComponentConfiguration getConfiguration() throws KuraException {
         final Tocd wiresOCD = new Tocd();
         wiresOCD.setId(CONF_PID);
-        wiresOCD.setName(s_message.name());
-        wiresOCD.setDescription(s_message.description());
+        wiresOCD.setName(message.name());
+        wiresOCD.setDescription(message.description());
 
         final Map<String, Object> props = CollectionUtil.newHashMap();
         for (final Map.Entry<String, Object> entry : this.properties.entrySet()) {
             final String key = entry.getKey();
-            final boolean isNotWireConfigurationProperty = !(key.endsWith(s_message.emitter())
-                    || key.endsWith(s_message.receiver()) || key.endsWith(s_message.filter()));
+            final boolean isNotWireConfigurationProperty = !(key.endsWith(message.emitter())
```

Comment:
I don't think it's a formatter issue but an user-specific configuration to use or not use any brackets.